pyuvsim/tests/test_source.py

```diff
@@ -83,11 +79,11 @@ def test_pol_coherency_calc():
     names = np.arange(Ncomp).astype('str')
     freq = np.ones(Ncomp) * 1e8 * units.Hz
 
-    stokes = np.zeros((4, Ncomp))
-    stokes[0, :] = 1.0
-    stokes[1, :Ncomp // 2] = 2.5
+    stokes = np.zeros((4, len(freq), Ncomp))
```

Comment:
This will add an axis to stokes with length `Ncomponents` (since `freq` is defined in the line above that way). Unless you coincidentally have the same number of sources as you do frequencies in the simulation, this shouldn't be the case.